public/javascripts/ng-controllers.js

```diff
@@ -108,3 +109,44 @@ springInitiative.controller('mainController', function($scope, $http) {
   };
 
 });
+
+springInitiative.controller('indexController', function($scope, $rootScope, $http, $location){
+    $scope.students = [];
+
+    // checkUser($rootScope, $location, $http);
+    $scope.user = $rootScope.loggedInUser;
+    console.log($rootScope.loggedInUser);
+
+    // TODO: make this cleaner using http.get(...)
+    $http({
+          method: 'GET',
+          url: '/api/allStudents'
+        })
+        .success(function(data){
+            console.log(data);
+            $scope.students = data;
+         })
+        .error(function(err){
+             console.log('Error: in GET \'/student\'', err);
+        }
+    );
+});
+
+springInitiative.controller('overviewController', function($scope, $rootScope, $http, $location){
+});
+
+springInitiative.controller('programController', function($scope, $rootScope, $http, $location){
+    $scope.programName = 'something spring';
+    $scope.programInfo = ':3';
+});
+
+springInitiative.controller('studentController', function($scope, $rootScope, $http, $location){
+    $scope.studentName = 'd';
+    $scope.studentInfo = 'we';
+});
+
+springInitiative.controller('addStudentController', function($scope, $rootScope, $http, $location){
+});
+
+springInitiative.controller('settingsController', function($scope, $rootScope, $http, $location){
+});
```

Comment:
What is all of this code/where did all of the controller for all of the student front-end go?  (We already have working controller code that handles add students, editing, etc.)  Also, why are we using a new controller for each functions?  Isn't that kind of redundant since this will all be happening on the same page?  I might be totally wrong here, but it seems like the code brenna and i worked on to get angular and the back end working with all students has been removed?  Idk, let me know or maybe it would help it someone could explain this to me tonight.